compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/platform/ParagraphBuilder.skiko.kt

```diff
@@ -160,7 +160,7 @@ private sealed interface ComputedStyle {
             fontStyle?.let {
```

Comment:
Please add Fixes "[CMP-8028](https://youtrack.jetbrains.com/issue/CMP-8028) Text color is sometimes randomly black on iOS when using Compose Multiplatform (BasicText)" in the description to update the issue

Reply:
It's not directly related to that, but ok